lib/git-fastclone.rb

```diff
@@ -189,18 +189,39 @@ def parse_inputs
       [url, path, options]
     end
 
+    def clear_clone_dest_if_needed(attempt_number, clone_dest)
+      return unless attempt_number.positive?
+
+      clone_dest_contents = Dir["#{clone_dest}/*"]
```

Comment:
Would this also catch hidden files (i.e. files prefixed with a `.`)? It seems unlikely to have only dot-files left in a clone directory, but maybe could happen especially if some other process/script cleaned out only `*` and thus left behind `.git` dirs, etc.?

Reply:
Good call, I'll double check

Reply:
You're right that it doesn't pick up hidden files, I'll push a fix.